lib/index.js

```diff
@@ -2,13 +2,10 @@
 
 import base64_url_decode from "./base64_url_decode";
 
-export function InvalidTokenError(message) {
-    this.message = message;
+export class InvalidTokenError extends Error {
+    name = "InvalidTokenError";
```

Comment:
no idea which browsers/versions of node you support (no CI or tests in this repo, and no `engines` field in package.json) - this can be done in the constructor if you support some old versions

Reply:
i.e.

```js
export class InvalidTokenError extends Error {
  constructor(message) {
    super(message);
    this.name = "InvalidTokenError";
  }
}
```